src/fragments/lib/interactions/react-native/frontend.mdx

```diff
@@ -0,0 +1,22 @@
+import reactnative1 from '/src/fragments/lib/in-app-messaging/integrate-your-application/react-native/configure-amplify.mdx';
+
+### Install Amplify and its dependencies
+
+```bash
+npm install aws-amplify @aws-amplify/react-native @aws-amplify/interactions @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values
+```
+
+You need to add the crypto.getRandomValues polyfills to your application's entry point file (in most React Native apps this will be the top level index.js).
```

Comment:
```suggestion
You need to add the `crypto.getRandomValues` polyfills to your application's entry point file (in most React Native apps this will be the top level index.js).
```